tests/bfd/test_bfd.py

```diff
@@ -304,7 +314,9 @@ def create_bfd_sessions_multihop(ptfhost, duthost, loopback_addr, ptf_intf, neig
     ptfhost.template(src='templates/bfd_responder.conf.j2', dest='/etc/supervisor/conf.d/bfd_responder.conf')
     ptfhost.command('supervisorctl reread')
     ptfhost.command('supervisorctl update')
-    ptfhost.command('supervisorctl start bfd_responder')
+    ptfhost.command('supervisorctl restart bfd_responder')
```

Comment:
for first run, wondering whether "restart" still work?

Reply:
Well tested, restart = start if not started.